plugins/cachix/auth_token_test.go

```diff
@@ -0,0 +1,53 @@
+package cachix
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func TestAuthTokenImporter(t *testing.T) {
+	plugintest.TestImporter(t, AuthToken().Importer, map[string]plugintest.ImportCase{
+		"Cache auth token": {
+			Environment: map[string]string{
+				"CACHIX_AUTH_TOKEN": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
```

Comment:
I like that you created a separate test for the personal auth token and the cache token 👌 

In general, we like the examples in the test to be as close to the real thing as possible. After playing a bit with Cachix, I noticed that these tokens are JWT's. With a bit of tinkering using jwt.io, I constructed two example tokens that don't actually work with Cachix, but do resemble the real deal: 

Cache auth token:
```
eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiI2MGI1ZTJlNy03ZDZiLTRiZGYtYjhiMS1mZGU2NDgzMmY5YzgiLCJzY29wZXMiOiJjYWNoZSJ9.PXNGrCN7ovMgEK0haz9voQfeECCwzzD79mEwg9KjqVE
```

Personal auth token: 
```
eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiI2MGI1ZTJlNy03ZDZiLTRiZGYtYjhiMS1mZGU2NDgzMmY5YzgiLCJzY29wZXMiOiJ0eCJ9.8u5huhMCEX8v58kp6oCU6ueJ8-EXzMNnpH5ERBKEabs
```

(you can paste them in https://jwt.io/ to see what makes them different)

Shall we use those in the tests?

Reply:
Thanks, makes sense to me. Done with [`803f6fa` (#97)](https://github.com/1Password/shell-plugins/pull/97/commits/803f6fa987def6487904e9620103167c537b6a6d).